monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/src/resource-pages/ApiDetailsPage.jsx

```diff
@@ -80,10 +82,25 @@ export default class ApiDetailsPage extends Component {
                         <TableBody>
                             {
                                 this.state.metaData.map(row => (
-                                    <TableRow>
-                                        <TableCell>{row.name}</TableCell>
-                                        <TableCell>{row.value}</TableCell>
-                                    </TableRow>
+
+                                    row.name == "Tracing" ?
+                                        (
+                                            <TableRow>
+                                                <TableCell>{row.name}</TableCell>
+                                                <TableCell>
+                                                    <Switch height={18} width={36}
```

Comment:
Don't we have these heights and widths defined as constants?

Reply:
will fix it separately.

Reply:
added in https://github.com/wso2/product-mi-tooling/pull/56